app/code/Magento/CatalogImportExport/Model/Import/Uploader.php

```diff
@@ -180,9 +180,9 @@ public function move($fileName, $renameFileOff = false)
             }
 
             $fileName = preg_replace('/[^a-z0-9\._-]+/i', '', $fileName);
-            $filePath = $this->_directory->getRelativePath($filePath . $fileName);
+            $tempPath = $this->_directory->getRelativePath($filePath . $fileName);
```

Comment:
Can you please use kind of "$relativePath" variable name? Its more clear then what is it doing

Reply:
@nuzil 
Yes sure

Reply:
@nuzil 
I have updated it in latest commit.
Thanks.